frontend/src/tests/pages/Courses/CoursesCreatePage.test.js

```diff
@@ -84,17 +84,15 @@ describe("CoursesCreatePage tests", () => {
 
         await waitFor(() => expect(axiosMock.history.post.length).toBe(1));
 
-        // expect(quarterField).toHaveValue("20124");
-        //expect(setQuarter).toBeCalledWith("20124"); //need this and axiosMock below?
-
         expect(axiosMock.history.post[0].params).toEqual(
             {
-            "psId": "13",
+            "psId": "",
```

Comment:
This is suspicious... why is this an empty string here?

I'll let it slide, but I think this might come back to bite you later.
so I want to encourage you to take a look